io/disk/ltp_fsstress.py.data/ltp_fsstress.yaml

```diff
@@ -1,7 +1,8 @@
-disk:
-script: 'fsstress'
-dir: '/mnt/nfs'
-fsstress_loop: 1
+disk: ''
+dir: ''
+fsstress_loop:
+nval: '250'
+pval: '250'
```

Comment:
Does readme explains the values to be given fsstress_loop:
nval: '250'
pval: '250'

Reply:
we do not have the Readme file. will add one with all required info